pinc/theme.inc

```diff
@@ -409,7 +409,7 @@ function html_navbar()
         echo "  &#183; ";
 
         $account_links = [
-            ['text' => _("Profile"), 'url' => "stats/members/mdetail.php"],
+            ['text' => _("Profile"), 'url' => "stats/members/mdetail.php?id=" . $user->u_id],
```

Comment:
That seems fine. I guess a question here is whether `stats/member/mdetails.php` should point to the logged in user by default? This would remove the need to set the userID through the URL's query.

Reply:
It does already, which is oddly the problem. The [`is_url_for_current_page()`](https://github.com/DistributedProofreaders/dproofreaders/blob/master/pinc/misc.inc#L1609) logic detects that the id-less page is a subset of the link and decides that it is the same page when it's actually not. We use that subset logic to make sure that the Round links correctly reflect that the user is on them.

Reply:
Sounds good, thanks for the explanation.